### PR TITLE
First pullRequest

### PR DESCRIPTION
**Pull request by Bronzza**

### CODE REVIEW

File: src/com/company/FlowerStore.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class FlowerStore {
+
+    private int wallet = 0;
+
+    public FlowerStore() {
```
Comment (Taniann) [inline src/com/company/FlowerStore.java]: it is default constructor

File: src/com/company/FlowerStore.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            }
+            if (i < tulip) {
+                flowers[counter] = new Tulip();
+                wallet = flowers[counter++].getPrice();
+            }
+        }
+        return flowers;
+    }
+}
+
```
Comment (Taniann) [inline src/com/company/FlowerStore.java]: remove redundant blank lines

File: src/com/company/FlowerStore.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    public Flower[] sellSequence(int rose, int chamomile, int tulip) {
+        Flower[] flowers = new Flower[rose + chamomile + tulip];
+        int counter = 0;
+        int maxNumber = Stream.of(rose, chamomile, tulip).max((p1, p2) -> (p1.compareTo(p2))).get();
+        for (int i = 0; i <  maxNumber; i++) {
+            if (i < rose) {
+                flowers[counter] = new Rose();
+                wallet = flowers[counter++].getPrice();
```
Comment (Taniann) [inline src/com/company/FlowerStore.java]: here you assign to wallet new value. check your wallet. now it returns wrong value

File: src/com/company/Flowers/Chamomile.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package com.company.Flowers;
+
+public class Chamomile extends Flower {
+
+    public Chamomile() {
+        price = 70;
+    }
+
+    public int getPrice() {
```
Comment (Taniann) [inline src/com/company/Flowers/Chamomile.java]: you already have the same method in parent class

File: src/com/company/Flowers/Flower.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package com.company.Flowers;
+
+public class Flower {
```
Comment (Taniann) [inline src/com/company/Flowers/Flower.java]: better make it abstract

File: src/com/company/Flowers/Rose.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
+package com.company.Flowers;
+
+public class Rose extends Flower {
+
+    public Rose() {
+        price = 100;
+    }
+
+    public int getPrice() {
```
Comment (Taniann) [inline src/com/company/Flowers/Rose.java]: the same

File: src/com/company/Flowers/Chamomile.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
+package com.company.Flowers;
+
+public class Chamomile extends Flower {
+
+    public Chamomile() {
+        price = 70;
```
Comment (Taniann) [inline src/com/company/Flowers/Chamomile.java:6]: for this case it is ok, but better to pass value as parameter instead of using "magic numbers"

**Review by Taniann — CHANGES_REQUESTED** [review]
please resolve comments

File: src/com/company/Main.java [hunk 8]
```diff
@@ -0,0 +1,23 @@
+package com.company;
+
+import com.company.Flowers.Flower;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class Main {
```
Comment (Taniann) [inline src/com/company/Main.java]: better name "FlowerStoreApp"

File: src/com/company/Flowers/Chamomile.java (hunk 7 above)
Comment (Bronzza) [reply src/com/company/Flowers/Chamomile.java:6]: Thanks, I understand, will try to avoid this approach.

**Review by Taniann — APPROVED** [review]
good

**Pull request merged**